src/detectLaunchConfigurationChanges.ts

```diff
@@ -0,0 +1,32 @@
+import { Workspace } from "./interfaces/Workspace";
+
+interface PromptRestartParams {
+  message: string;
+  reloadWindowChoice: string;
+  dismissChoice: string;
+}
+
+export function detectLaunchConfigurationChanges(
```

Comment:
Just a a thought, but should we also provide an easy way to add another key that may be unique to the client? For example, maybe there is one relevant to `coc-metals` but not for VS Code. It's be great to have another optional parameter in here to add another key. Thoughts?

Reply:
I thought about it, but since this is extremely specific to our two clients and we can iterate really quickly, I figured we could add it once we need it.

What do you think?

(By the way, I should give you publishing rights and also setup automation for publishing via the CI)

Reply:
Yea, I suppose that's true. Totally fine the way it is then.